src/scripts/domParsers/domUtils.ts

```diff
@@ -372,6 +371,8 @@ export module DomUtils {
 				theImg.onerror = () => {
 					resolve(undefined);
 				};
+
+				theImg.src = img.src;
```

Comment:
Do you think that it's worth a comment here noting the importance of setting the src property _after_ everything else?